src/transaction/log_page_buffer.c

```diff
@@ -4499,10 +4499,17 @@ logpb_flush_pages (THREAD_ENTRY * thread_p, const LOG_LSA * flush_lsa)
       // *INDENT-OFF*
       if (get_server_type () == SERVER_TYPE_TRANSACTION && ts_Gl->is_page_server_connected ())
 	{
-	  log_Gl.wait_flushed_lsa (*flush_lsa);
 	  if (prm_get_bool_value (PRM_ID_ER_LOG_COMMIT_CONFIRM))
 	    {
-	      _er_log_debug (ARG_FILE_LINE, "Page server committed LSA = %lld|%d.\n", LSA_AS_ARGS (&log_Gl.m_max_ps_flushed_lsa));
+	      _er_log_debug (ARG_FILE_LINE, "[COMMIT CONFIRM] Wait until consenesus LSA gets larger than or equal to %lld|%d.\n", LSA_AS_ARGS (flush_lsa));
```

Comment:
logpb_flush_pages () is called by the checkpoint function.
(ATS checkpoint : logpb_checkpoint_trantable () -> logpb_flush_pages () )

When ATS do checkpoint, I think this debug log message can be logged. 

if the checkpoint also need the consensus, how about just removing `[COMMIT CONFIRM]`  ?

Reply:
It makes sense. While considering your suggestion, a concern came up that some regression tests would use the existing debug logging, and it turned out to be the case. So, I'll leave existing errors as it is, and just add some logging for this PR without "COMMIT CONFIRM". Thanks.